src/pages/ClusterPage/utils/index.js

```diff
@@ -342,30 +342,40 @@ export function deployMyCluster(body) {
 }
 
 export function verifyCluster(url) {
-	const { origin, username, password } = new URL(url);
-	let headers = {};
-
-	if (username && password) {
-		headers = {
-			Authorization: `Basic ${btoa(`${username}:${password}`)}`,
-		};
-	}
 	return new Promise((resolve, reject) => {
-		fetch(origin, {
-			method: 'GET',
-			headers,
+		fetch(`${ACC_API}/v1/_verify_es_connection`, {
+			method: 'POST',
+			credentials: 'include',
+			headers: {
+				'Content-Type': 'application/json',
+			},
+			body: JSON.stringify({
+				elasticsearch_url: url,
+			}),
 		})
-			.then(res => {
-				if (res.status === 401 || res.status === 403) {
-					throw new Error('Auth Error');
-				}
-				return res.json();
-			})
+			.then(res => res.json())
 			.then(data => {
-				resolve(data);
+				if (
+					data.version &&
+					parseInt(data.version.number.split('.')[0], 10) < 6
+				) {
+					reject(new Error('Arc is not supported for version < 5'));
```

Comment:
@lakhansamani Error message doesn't match parse int. `<= 6.x` should be the message. Also, use the word `appbase.io` instead of Arc.

Reply:
Fixing